evap/results/exporters.py

```diff
@@ -215,15 +232,18 @@ def write_overall_results(self, evaluations_with_results, course_results_exist):
 
         if course_results_exist:
             # Only query the number of evaluations once and keep track of it here.
-            count_gt_1 = [e.course_evaluations_count > 1 for e in evaluations]
+            count_gt_1 = [e.course_evaluations_count > 1 for e in evaluations]  # type: ignore[attr-defined]
```

Comment:
Again, can we annotate `evaluations` above once to make the code using the monkey-patched attributes work without additional type annotations?

Reply:
I dont have any idea